src/translations/en.ts

```diff
@@ -8,5 +8,12 @@ export default {
     "Updated Book Details": "Updated Book Details",
     "Submit": "Submit",
     "Search" : "Search",
+    "Add New Book" : "Add New Book",
```

Comment:
please correct all keys in translations, all keys should be in lower case and without any spaces - instead of spaces you should use dash "-"

Reply:
Done